common/api/core-frontend.api.md

```diff
@@ -7048,10 +7048,12 @@ export interface MaterialSpecularProps {
 
 // @public
 export interface MaterialTextureMappingProps {
+    constantLodParams?: TextureMapping.ConstantLodParamProps;

```

Comment:
I think this was renamed to constantLodProps.

Reply:
@MarcNeely must not have rerun api extractor.

Reply:
I reran it.